### PR TITLE
Refactor messages with client/customer/person to Client/Customer

### LINKED ISSUE #109: Refactor messages with client/customer/person to Client/Customer

### PR DESCRIPTION
**Pull request by colinhia**
Closes #109

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
     public static final String MESSAGE_SUCCESS = "AgentAssist's contacts have been cleared!";
-    public static final String MESSAGE_CLEAR_CONFIRMATION = "This will permanently clear all contacts. "
+    public static final String MESSAGE_CLEAR_CONFIRMATION = "This will permanently clear all clients' contacts. "
```
Comment (iamdiluxedbutcooler) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java:15]: looks good & clear!

**Review by iamdiluxedbutcooler — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/ClearCommand.java (hunk 1 above)
Comment (ZShunRen) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java:15]: nice!

**Review by ZShunRen — APPROVED** [review]
LGTM

**Review by itsme-zeix — APPROVED** [review]
LGTM!

**Pull request merged**